src/overlays/actors/ovl_En_Diving_Game/z_en_diving_game.c

```diff
@@ -9,11 +17,29 @@ void EnDivingGame_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnDivingGame_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnDivingGame_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_0600219C;
-extern UNK_TYPE D_06002FE8;
-extern UNK_TYPE D_0600BFA8;
+void func_809EDCB0(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EDD4C(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EDD4C(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EDEDC(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE048(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE0FC(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE194(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE1F4(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE1F4(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE408(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE6C8(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE780(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE800(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE8F0(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EE96C(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EEA00(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EEA90(EnDivingGame* this, GlobalContext* globalCtx);
+void func_809EEAF8(EnDivingGame* this, GlobalContext* globalCtx);
+s32 func_809EEDE4(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx);
+void EnDivingGame_SpawnRuppy(EnDivingGame* this, GlobalContext* globalCtx);
+s32 func_809EDB08(EnDivingGame* this, GlobalContext* globalCtx);
+Gfx* EnDivingGame_EmptyDList(GraphicsContext* gfxCtx);
```

Comment:
I think a lot of these functions don't need to be forward declared so they can be omitted